NetworkManager.go

```diff
@@ -203,6 +203,9 @@ type NetworkManager interface {
 	// Indicates whether connectivity checking is enabled. This property can also be written to to disable connectivity checking (as a privacy control panel might want to do).
 	GetPropertyConnectivityCheckEnabled() (bool, error)
 
+	// NetworkManager Settings
+	GetPropertySettings() (Settings, error)
```

Comment:
I'm not sure you wanted to add this. It's not related to your PR description and it's not implemented by networkManager.

Reply:
Therefore any usage of NewNetworkManager() will fail at build.